docs/examples/utility/add_text_example.py

```diff
@@ -0,0 +1,36 @@
+"""
+add_text Example
+================
+
+Multiple examples of combinations of x and y aliases for the add_text function.
+"""
+
+###
+from plothist import add_text
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots()
+
+positions = [
+    ("right_in", "top_in"),
+    ("left_in", "top_in"),
+    ("left_in", "bottom_in"),
+    ("right_in", "bottom_in"),
+    ("right", "top_out"),
+    ("left", "top_out"),
+    ("right_out", "top_in"),
+    ("right_out", "bottom_in"),
+    ("right", "bottom_out"),
+    ("left", "bottom_out"),
+]
+
+for x, y in positions:
+    x_label = x.replace("_", "\_")
+    y_label = y.replace("_", "\_")
+    add_text(
+        f"$\\mathtt{{add\\_text()}}$\n$\\mathtt{{x = {x_label}}}$\n$\\mathtt{{y = {y_label}}}$",
```

Comment:
Add quotes around the arguments to show that they are string, as in 

https://plothist.readthedocs.io/en/latest/advanced/model_examples.html#all-the-different-comparisons